addons/crm/models/crm_lead.py

```diff
@@ -1338,11 +1338,24 @@ def action_show_potential_duplicates(self):
         }
         return action
 
-    def action_snooze(self):
```

Comment:
Same as in mail, don't remove code in stable, in case people pull code but don't update views. Just note it is deprecated.